cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -82,4 +82,6 @@ private CruiseControlConfiguration(String configuration, List<String> forbiddenO
     public static CruiseControlConfiguration unvalidated(String string) {
         return new CruiseControlConfiguration(string, emptyList());
     }
+
+
```

Comment:
removing this two more blank lines will avoid to have this file changed :-)

Reply:
Whoops, I'll fix that!